src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -584,6 +587,18 @@ public void SetNonBlocking()
             }
         }
 
+        public void CheckForPriorConnectFailure()
+        {
+            if (_connectFailed)
```

Comment:
+!.  yes, please put braces around that in the Allman style as per CoreFx coding guidelines.